desktop-icon.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/misc/ncspot.desktop b/misc/ncspot.desktop
```

Comment:
no need for a patch. you can just use `desktop-file-edit`

`rename-icon` also take care of it but you don't use it here.